### ISSUE TITLE
how to add voice data to Osmand Local Indexes List

### ISSUE THREAD

**Issue by herotux** [osmandapp/OsmAnd#5659]
i want add this voice guide file to [Osmand Local Indexes List ](http://download.osmand.net/list.php)

this file made by iranian osm editors and we like see them in new version of osm thanks .
[fa_0.voice.zip](https://github.com/osmandapp/Osmand/files/2181245/fa_0.voice.zip)

**Comment by sonora** [comment]
I am not currently aware how to do this, hence will have to wait on someone else in the dev  team to pick this up.

But I just notice there is already also another recoded Farsi voice present at https://github.com/osmandapp/OsmAnd-resources/tree/master/voice/fa which also does not seem available as a download (or am I wrong?) So to whoever picks this up: It looks like there are more occurrences of existing recorded voices in other languages we do not make available, and  this needs an overall cleanup?

**Comment by PaulStets** [comment]
I created a pull request with the new files: https://github.com/osmandapp/OsmAnd-resources/pull/494/files

**Comment by sonora** [comment]
Paul, unless I am mistaken, that pull request would overwrite the already existing Persian recorded voice present in our code, and I am not sure that was the original intent in this issue? From what I see, the existing voice seems neither offered under Ddownloads nor can it be selected in the OsmAnd voice guidance menu (it is missing under 'Install more...), as are several other recorded voices we have. That to me seems worth fixing primarily? And in the process we could simply add this new fa voice here as an additional variant for testing?

**Comment by PaulStets** [comment]
@sonora, I didn't know the files were missing in the app, sorry. I simply created the request, because a user asked me to.

**Comment by sonora** [comment]
Yes, my issue is I am currently uncertain what scripts and mechanism we may or may not have linking voices we have in our code (on github) vs.  which are made available for download from within the app. Once that is understood (and maybe fixed) we can easily add this new voice as an additional version to the correct place.

**Comment by vshcherb** [comment]
Well for all "recorded" voices we need to package them and upload to the server, I guess at one point we stopped doing it because for most of the languages TTS reached a good level enough.

**Comment by sonora** [comment]
Yes, fully agree we should promote TTS in all cases. I have no idea about Farsi, though, maybe lack of TTS sipport here justifies an exception.

**Comment by vshcherb** [comment]
I think if the attached file could be confirmed that has reasonable quality and tested, we could upload it to the server downloads.
